src/components/Chat/MembersSider/Member/index.js

```diff
@@ -10,6 +11,25 @@ const Member = props => {
     return profiles && profiles[userId] && profiles[userId].username
   }
 
+  if (isAdmin(userId, profiles)) {
+    return (
+      <div className="chat-header-members-sider-member">
+        <div>
+          <div
+            className="label chat-message-sender-icon"
+            style={{
+              backgroundColor: 'black',
+              color: 'white',
+            }}
+          >
+            K
```

Comment:
Why K? Maybe it's for Kohdataan. I guess this will eventually be replaced with a logo or something.